sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingJoinSuite.scala

```diff
@@ -76,7 +76,7 @@ abstract class StreamingJoinSuite
     (input1, input2, select)
   }
 
-  protected def setupWindowedJoinWithLeftCondition(joinType: String)
+  protected def setupJoinWithLeftCondition(joinType: String)
```

Comment:
Here as well. Please revisit your changes to apply my comment instead of fixing only where I pointed out.

Reply:
Sorry for missing these two at the first place. Updated.